### PR TITLE
Improve API

### PR DESCRIPTION
**Pull request by icemac**
Alias ``.browser.urllib_request.HTTPError`` to ``.browser.HTTPError`` to have a better API.

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -2,10 +2,11 @@
 CHANGES
 =======
 
-5.0.1 (unreleased)
-------------------
+5.1 (unreleased)
```
Comment (mgedmin) [inline CHANGES.rst]: Minor: We had a 5.0.0, so maybe make this 5.1.0 rather than 5.1?

File: src/zope/testbrowser/browser.py [hunk 2]
```diff
@@ -281,6 +282,7 @@ def _processRequest(self, url, make_request):
                     resp = self.testapp.get(url, **reqargs)
             assert remaining_redirects > 0, "redirects chain looks infinite"
             self._setResponse(resp)
+            HTTPError = urllib_request.HTTPError
```
Comment (mgedmin) [inline src/zope/testbrowser/browser.py]: This local variable assignment seems pointless?

File: CHANGES.rst (hunk 1 above)
Comment (icemac) [inline CHANGES.rst]: I am constantly working to get rid of these feature version numbers ending in `.0`

In my opinion the version number of the unreleased version should always be the next feature version as we are developing towards the next feature not towards the next bug fix
Using `zest.releaser` for the release always requires manual changes when using a version number consisting of three parts.

File: src/zope/testbrowser/browser.py (hunk 2 above)
Comment (icemac) [inline src/zope/testbrowser/browser.py]: Nice catch I am not sure how this one snitched in.

**Pull request merged**